backend/routes/auth.py

```diff
@@ -3,16 +3,16 @@
 from steam.api import get_steam_id, get_player_summary
 import bcrypt
 auth_blueprint = Blueprint('auth', __name__)
-
+from database.util import find_user_by_email, verify_password
```

Comment:
consider doing `*` for all future imports
```suggestion
from database.util import *
```